src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioPreviewTest.java

```diff
@@ -75,17 +75,6 @@ void audioPreviewEventuallyBecomesReady() {
         });
     }
 
-    @Test
-    void audioPreviewEventuallyFailsWhenMediaDoesntExist() {
```

Comment:
Author note: This test was removed as it's not actually possible to hit this scenario. A previous change carried out in DMP-4304 checks media exists and if not throws an error. This was done in the controller layer, and therefore this service layer test was not picked up as part of that change. I've moved the validation to the service layer (AudioPreview) as believe controllers should be kept as skinny as possible and any real logic should not be in controllers. This caused this test to fail as this class now checks that media exists. As we have a test covering this scenario in the AudioControllerPreviewIntTest, this can be safely removed.